### PR TITLE
Configure logging explicitly, instead of relying on `Environment`

### PR DESCRIPTION
**Pull request by kim**
Deploy using debug logging and no layout + colour.

I left the defaults at colourful logging for convenience, at the expense of CLI options and logger defaults no differing.

### CODE REVIEW

File: src/Oscoin/Telemetry/Options.hs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+           <> metavar (enumBounded (Proxy @Log.LayoutFormat))
+           <> value   Log.HumanReadable
+           <> showDefault
+            )
+
+    enumBounded
+        :: forall proxy a. (Show a, Enum a, Bounded a)
+        => proxy a
+        -> String
+    enumBounded _ = intercalate "|" $ map (show @a) [minBound..maxBound]
```
Comment (adinapoli-mndc) [inline src/Oscoin/Telemetry/Options.hs]: This looks like a generally-useful function, I would be in favour of having it at the top level 😉

Comment (kim) [reply src/Oscoin/Telemetry/Options.hs]: Sure... but ... where?

Comment (adinapoli-mndc) [reply src/Oscoin/Telemetry/Options.hs]: I do wonder if something _like Oscoin.Prelude_ (but perhaps not the prelude directly) might be a good candidate 🤔

Comment (kim) [reply src/Oscoin/Telemetry/Options.hs]: I don't think this is in any way a prelude-ish function, it applies specifically to `optparse-applicative` `metavar`s, where you'd want to display the available options from an enum. If you can think of more options-related things, that may justify some options utils fns module.

**Pull request merged**